easy_format.ml

```diff
@@ -57,12 +65,14 @@ let list = {
 }
 
 type label_param = {
+  label_break: label_break;
   space_after_label : bool;
   indent_after_label : int;
   label_style : style_name option;
 }
 
 let label = {
+  label_break = `Auto;
```

Comment:
I'm realizing that having functions with named args/defaults be the primary way to construct them - might allow evolving of the API in a backwards compatible way. Just a realization - not important.